net/siproxd/src/opnsense/scripts/OPNsense/Siproxd/setup.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+
+mkdir -p /var/run/siproxd
+chown -R nobody:nogroup /var/run/siproxd
+chmod 750 /var/run/siproxd
+
+mkdir -p /var/lib/siproxd
+chown -R nobody:nogroup /var/lib/siproxd
+chmod 750 /var/lib/siproxd
```

Comment:
nice work pinning these. freebsd does not seem to list them: https://github.com/opnsense/ports/blob/47aa039c8c8f361193e18a636760392971ef57ae/net/siproxd/pkg-plist